app/src/main/res/layout/welcome_slide_1.xml

```diff
@@ -8,36 +8,53 @@
 
     <ImageView
         android:id="@+id/slide_2_image"
-        android:layout_width="@dimen/img_width_height"
-        android:layout_height="@dimen/img_width_height"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintWidth_percent="0.6"
+        app:layout_constraintDimensionRatio="2:1"
         android:layout_centerHorizontal="true"
-        app:srcCompat="@drawable/ic_nutriscore_c" />
+        android:contentDescription="TODO"
```

Comment:
can we remove it and add suppress lint fields or replace it with `android:contentDescription="@null"` ?